### DISCUSSION TITLE
我win11用CMD启动后闪退， 没有任务提示。 也没LOG。

### DISCUSSION THREAD

**Discussion by manalore** [AlistGo/alist#3455] [General]
win11用CMD启动后闪退， 没有任务提示。 也没LOG。这是怎么回事啊？？？

**Comment by Dashan-IZ** [comment]
你可以加qq群向大佬提问哦，大佬多多，有问必答，qq群号：746260284，

还有你这个问题，如果你是x64的电脑，建议你下载amd64的alist包，然后解压，用pwershell打开用   .\alist.exe start 运行，要保证命令窗口打开，关闭命令窗口的话，alist也会一起关闭，本地登录的话直接127.0.0.1：5244就可以打开，多看看文档
